requirements.yml

```diff
@@ -2,9 +2,13 @@ collections:
   - name: community.general
     version: 8.6.0
   - name: scale_computing.hypercore
-    version: 1.5.0  # url2template role is needed
+    version: 1.5.0
   - name: kubernetes.core
     version: 3.0.1
+  - name: https://github.com/k3s-io/k3s-ansible.git
+    type: git
+    version: master
+    # version: 1907e6fb79fedcc38fe3be88b933b1643a9c9716  # commit we tested with
```

Comment:
I would use a tag here, but they do not have tags. So I just used master branch, and mentioned commit SHA for reference.